README_CHUNK_FORMAT.rst

```diff
@@ -125,6 +125,11 @@ for encoding blocks with a filter pipeline::
     to their index in increasing order. The type of filter applied is specified by the `filter_code`. Each
     `filter_code` has an associated field in `filter_meta` that can contain metadata about the filter.
 
+:compcode_meta:
+    (``uint8``) Compression codec metadata.
+
+    Possible metadata associated with the compression codec.
```

Comment:
Better rephrase this as:  "Metadata associated to User Defined Codec.  0 if not User Defined Codec."

Reply:
I rephrase the sentence as: "Possible metadata associated with the compression codec. Only used in user-defined codecs."

Additionally, a sentence like "See more here" can be added when a README_PLUGINS.md is redacted.

What do you think?

Reply:
Ok. I'd just remove the 'Possible' which does not bring siginificant meaning.

Reply:
Fixed